README.md

```diff
@@ -35,7 +35,7 @@ import { loadConnect } from "@stripe/connect-js";
 const stripeConnect = await loadConnect();
 const connectInstance = stripeConnect.initialize({
   publishableKey: "{{pk test123}}",
-  clientSecret: "{{client secret}}"
```

Comment:
The example in this repo still needs to be updated right?

Reply:
Yup, am going to update the readme in a separate PR